utilities/test_suite/HIP/Tensor_audio_hip.cpp

```diff
@@ -44,7 +44,7 @@ int main(int argc, char **argv)
     string scriptPath = argv[7];
 
     // validation checks
-    if (testType == 0 && batchSize != 3)
+    if (!testType && batchSize != 3) // unit test mode
```

Comment:
Can we take string for test_type instead of 0/1? Here we can check test_type against string ("QA") or can use enum

Reply:
done